src/test/test-unit-file.c

```diff
@@ -111,17 +111,30 @@ static void test_config_parse_exec(void) {
 
         ExecCommand *c = NULL, *c1;
         const char *ccc;
+        Manager *m = NULL;
+        Unit *u = NULL;
+
+        r = manager_new(MANAGER_USER, true, &m);
+        if (MANAGER_SKIP_TEST(r)) {
+                printf("Skipping test: manager_new: %s\n", strerror(-r));
```

Comment:
> Origin codes in test-unit-file.c were also using printf(). So I added printf() instead of log_.

Ah, ok.

> Do you want to change them all?

Yes please, but that can be done in a separate commit.